src/containerapp/HISTORY.rst

```diff
@@ -6,6 +6,8 @@ upcoming
 ++++++
 * 'az containerapp session code-interpreter execute': Extend maximum supported value of `--timeout-in-seconds` from 60 to 220.
 * 'az containerapp job create': Fix message with `--help`
+* 'az containerapp update': Disallow changing `--revisions-mode` to Labels.
```

Comment:
@Tratcher  please resolve conflict for this file. We already have 1.2.0b3 released. The file is out of date.